Doc/library/ast.rst

```diff
@@ -368,10 +368,19 @@ The following options are accepted:
    Specify what kind of code must be compiled, like the *mode* argument
    in :func:`parse`.
 
+.. cmdoption:: --no-type-comments
+
+   Don't parse type comments.
```

Comment:
You have to document that it's new, using ``.. versionadded:: 3.9`` for example (indent it properly, check usage in another documentation file).

Reply:
I dont think we need to specify it, this part of the document already covered by `.. versionadded:: 3.9` https://github.com/python/cpython/commit/832e8640086ac4fa547c055a72929879cc5a963a#diff-37ef1f6c2c9a9c8e36b33fa081e9f5e8R345-R348

Reply:
Oh, right.